test/json/v1beta1_api_resource_list.json

```diff
@@ -0,0 +1,11 @@
+{
+  "kind": "APIResourceList",
+  "groupVersion": "extensions/v1beta1",
```

Comment:
nope, I didn't add the whole API list. Just to make the test pass. I can create the total `v1beta1` list.